### PR TITLE
ASUS.GPUTweakIII version 1.5.8.0

### PR DESCRIPTION
**Pull request by DeusMaximus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98942)

### CODE REVIEW

**Comment by Trenly** [comment]
@denelon - I think this may be a bug with CLI; 3010 should be treated as a success code, not a fail code by default?

```
--> Installing WinGet
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Admin setting enabled.Admin setting enabled.

--> Installing the Manifest 1.5.8.0

Found GPU Tweak III Install [ASUS.GPUTweakIII] Version 1.5.8.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dlcdnets.asus.com/pub/ASUS/Graphic%20Card/Unique_Applications/GPU-Tweak-III-v1580.zip
  ██████████████████████████████  68.7 MB / 68.7 MB
Successfully verified installer hash
Extracting archive...
Successfully extracted archive
Starting package install...
Installer failed with exit code: 3010
Installer log is available at: C:\Users\WDAGUtilityAccount\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir\WinGet-ASUS.GPUTweakIII.1.5.8.0-2023-03-09-08-55-49.461.log
Restart your PC to finish installation.

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName                                                    DisplayVersion Publisher             ProductCode
-----------                                                    -------------- ---------             -----------
Microsoft Visual C++ 2017 x64 Minimum Runtime - 14.11.25325    14.11.25325    Microsoft Corporation {B0037450-526D-3448-A370-CACBD87769A0}
Microsoft Visual C++ 2017 x64 Additional Runtime - 14.11.25325 14.11.25325    Microsoft Corporation {B13B3E11-1555-353F-A63A-8933EE104FBD}
Microsoft Visual C++ 2017 x86 Minimum Runtime - 14.11.25325    14.11.25325    Microsoft Corporation {029DA848-1A80-34D3-BFC1-A6447BFC8E7F}
GPU Tweak III Install                                          1.5.8.0        ASUS                  {0d3769a0-fd52-4693-8744-4671f017952f}
Microsoft Visual C++ 2017 Redistributable (x86) - 14.11.25325  14.11.25325.0  Microsoft Corporation {404c9c27-8377-4fd1-b607-7ca635db4e49}
Microsoft Visual C++ 2017 x86 Additional Runtime - 14.11.25325 14.11.25325    Microsoft Corporation {568CD07E-0824-3EEB-AEC1-8FD51F3C85CF}
Microsoft Visual C++ 2017 Redistributable (x64) - 14.11.25325  14.11.25325.0  Microsoft Corporation {6c6356fe-cbfa-4944-9bed-a9e99f45cb7a}
GPU Tweak III                                                  1.5.8.0        ASUS                  {8753C623-2580-4C38-A5C6-229FF2A940A3}
```

File: manifests/a/ASUS/GPUTweakIII/1.5.8.0/ASUS.GPUTweakIII.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: ASUS.GPUTweakIII
+PackageVersion: 1.5.8.0
+Installers:
+- InstallerUrl: https://dlcdnets.asus.com/pub/ASUS/Graphic%20Card/Unique_Applications/GPU-Tweak-III-v1580.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: burn
+  NestedInstallerFiles:
+  - RelativeFilePath: GPU Tweak III Install.exe
+  InstallerSha256: 25A0D59345C039C581D7B670F23E9572937EA5D3E94610AF1CFF9B907A2D4642
```
Comment (stephengillie) [inline manifests/a/ASUS/GPUTweakIII/1.5.8.0/ASUS.GPUTweakIII.installer.yaml:13]:
```suggestion
  InstallerSha256: 25A0D59345C039C581D7B670F23E9572937EA5D3E94610AF1CFF9B907A2D4642
  InstallerSuccessCodes: 3010
```

**Comment by stephengillie** [comment]
Manifest validation error: 
>  Manifest Error: Schema Error [Schema validation failed. 
> Error context: <root>[Installers][0][InstallerSuccessCodes] Description: Value type not permitted by 'type' constraint. 
> Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'InstallerSuccessCodes'. 
> Error context: <root>[Installers] Description: Failed to validate item #0 in array. 
> Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] 
> File: ASUS.GPUTweakIII.installer.yaml Manifest Error: Failed to process field. [InstallerSuccessCodes]

File: manifests/a/ASUS/GPUTweakIII/1.5.8.0/ASUS.GPUTweakIII.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  InstallerSuccessCodes: 3010
```
Comment (stephengillie) [inline manifests/a/ASUS/GPUTweakIII/1.5.8.0/ASUS.GPUTweakIII.installer.yaml:14]:
```suggestion
  InstallerSuccessCodes: 
  - 3010
```

Comment (stephengillie) [reply manifests/a/ASUS/GPUTweakIII/1.5.8.0/ASUS.GPUTweakIII.installer.yaml:14]: Apologies - I failed to test the previous suggestion. This has been tested to ensure it works.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**